composer.json

```diff
@@ -17,7 +17,7 @@
     ],
     "require": {
         "php": "^8.0",
-        "illuminate/database": "^8.0",
+        "illuminate/database": "^8.76.1",
```

Comment:
```
sail@f34348e1ecf9:/var/www/html$ composer require spatie/laravel-sluggable
Using version ^3.2 for spatie/laravel-sluggable
./composer.json has been updated
Running composer update spatie/laravel-sluggable
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires spatie/laravel-sluggable ^3.2 -> satisfiable by spatie/laravel-sluggable[3.2.0].
    - spatie/laravel-sluggable 3.2.0 requires illuminate/database ^8.76.1 -> found illuminate/database[v8.76.1, ..., 8.x-dev] but these were not loaded, likely because it conflicts with another require.


Installation failed, reverting ./composer.json and ./composer.lock to their original content.

```

Reply:
What other packages do you use? And which version do you use for your project?

Reply:
![image](https://user-images.githubusercontent.com/49227202/147582094-8cd14cfb-7780-4b31-961d-e5d16cfdb06c.png)
I was able to install only after updating all packages